code/game/gamemodes/blob/theblob.dm

```diff
@@ -87,7 +88,10 @@
 		update_icon()
 
 /obj/effect/blob/proc/Life()
-	//writepanic("[__FILE__].[__LINE__] ([src.type])([usr ? usr.ckey : ""])  \\/obj/effect/blob/proc/Life() called tick#: [world.time]")
+	if(stun_time)
+		stun_time--
+	if(brute_resist<4 && prob(20))

```

Comment:
usually a good idea to wrap clauses (ie ```if((brute_resist < 4) && prob(20))```), and afaik the byond compiler doesn't use operator spacing to resolve order